src/Place/Canonical/DBALDuplicatePlaceRepository.php

```diff
@@ -147,4 +147,32 @@ public function addToDuplicatePlaces(PlaceWithCluster $clusterRecordRow): void
             ])
             ->execute();
     }
+
+    public function howManyPlacesAreToBeImported(): int
+    {
+        // COUNT from `duplicate_places_import` not present in `duplicate_places`
+        $result = $this->connection->createQueryBuilder()
+            ->select('COUNT(*) AS not_in_duplicate')
+            ->from('duplicate_places_import', 'dpi')
+            ->leftJoin('dpi', 'duplicate_places', 'dp', 'dpi.cluster_id = dp.cluster_id AND dpi.place_uuid = dp.place_uuid')
+            ->where('dp.cluster_id IS NULL')
+            ->execute();
+
+        $a = $result->fetchOne();
+
+        return (int)($a ?? 0);
+    }
+
+    public function howManyPlacesAreToBeDeleted(): int
```

Comment:
Can you not reuse `getClustersToBeRemoved`

Reply:
I did

Reply:
No, because that gives an amount of clusters, not places. The other number in the script is also related to places.
I did change it use "getPlacesNoLongerInCluster" and removed "howManyPlacesAreToBeDeleted"